catalyst-gateway/bin/src/service/api/documents/put_document/mod.rs

```diff
@@ -67,15 +67,15 @@ pub(crate) async fn endpoint(doc_bytes: Vec<u8>) -> AllResponses {
             };
 
             let doc_body = SignedDocBody::new(
-                doc.doc_id(),
-                doc.doc_ver(),
-                doc.doc_type(),
+                doc.doc_id().into(),
```

Comment:
Why did this change in this PR?

Reply:
An API of the catalyst-signed-doc was changed, in particular for these fields an return type of `uuid:Uuid` was changed to `catalyst-types::UuidV4`, `catalyst-types::UuidV7`.